lib/core/connector_settings.rb

```diff
@@ -13,13 +13,15 @@
 
 module Core
   class ConnectorSettings
+    # Error Classes
+    class ConnectorNotFoundError < StandardError; end
+
     def self.fetch(connector_id)
-      es_response = ElasticConnectorActions.load_connector_settings(connector_id)
-      if es_response['found'] == false
-        Utility::Logger.debug("Connector settings not found for connector_id: #{connector_id}")
-        return nil
-      end
-      new(es_response.with_indifferent_access)
+      es_response = ElasticConnectorActions.get_connector(connector_id)
+        .with_indifferent_access
```

Comment:
duplicate `with_indifferent_access`

Reply:
I deleted it from line 22, so it should not be duplicate any more